### PR TITLE
Update travis

### PR DESCRIPTION
**Pull request by mirpo**

### CODE REVIEW

**Review by juslesan — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
+- 13.9.0
+git:
+  depth: false
 branches:
   only:
-  - master
-  - "/^v\\d+\\.\\d+(\\.\\d+)?(-\\S*)?$/"
+    - master
+    - "/^v\\d+\\.\\d+(\\.\\d+)?(-\\S*)?$/"
+    - update_uws
```
Comment (harbu) [inline .travis.yml]: Do we still need `update_uws`?

File: .travis.yml [hunk 2]
```diff
@@ -1,13 +1,15 @@
 language: node_js
 node_js:
-- 10.17.0
-sudo: true
+- 13.9.0
+git:
+  depth: false
```
Comment (harbu) [inline .travis.yml]: ?

File: Dockerfile [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # Use official node runtime as base image
-FROM node:10.17.0-alpine
+FROM node:13
```
Comment (harbu) [inline Dockerfile]: Btw... I think this Dockerfile can be removed since we are only using broker now?

File: README.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ base.
 
 ## Installation
 
-Prerequisites: [Node.js](https://nodejs.org/) (`^8.10.0`, `^10.13.0`, or `>=11.10.1`), npm version 6+.
+Prerequisites: [Node.js](https://nodejs.org/) (`>=^10.13.0`), npm version 6+.
```
Comment (harbu) [inline README.md]: Should we say exact version because that is what we are actually testing: 13.9.0

File: Dockerfile (hunk 3 above)
Comment (mirpo) [reply Dockerfile]: Ok

File: .travis.yml (hunk 2 above)
Comment (mirpo) [reply .travis.yml]: I'll remove

File: .travis.yml (hunk 1 above)
Comment (mirpo) [reply .travis.yml]: No, it's only for this branch. I'll remove

File: README.md (hunk 4 above)
Comment (mirpo) [reply README.md]: Ok

**Pull request merged**